app/helpers/compute_resources_helper.rb

```diff
@@ -43,8 +43,10 @@ def memory_options(max_memory)
     opts.map{|n| [number_to_human_size(n*Foreman::SIZE[:giga]), (n*Foreman::SIZE[:giga]).to_i] unless n > (max_memory / Foreman::SIZE[:giga])}.compact
   end
 
-  def password_placeholder(obj)
-    obj.id ? "********" : ""
+  def password_placeholder(obj, attr = nil)
+    pass ||=  !obj.read_attribute(attr).empty? || obj.has_attribute?(:password_hash)
```

Comment:
Did a,b,c.
Regarding to d, it is a workaround for the User model, but anyway it makes sense not just for it, in the DB the password will be saved under password or under password_hash(for sensitive passwords).